### DISCUSSION TITLE
blueprint vs drawing?

### DISCUSSION THREAD

**Discussion by tve** [sgenoud/replicad#91] [Q&A] [answered]
What's the difference between a blueprint and a drawing? Or what can one do with a blueprint that one can't do with a drawing?

**Comment by sgenoud** [comment, marked as answer]
The drawing is a nicer API on top of blueprints (which is a concept I'd like to make disappear when I start using [pantograph](https://github.com/sgenoud/pantograph) for the 2D layer of replicad.

When you fuse two blueprints you can get an array of blueprint. When you cut them you can have a compound blueprint (a blueprint within a blueprint). Any of the boolean operations on a drawing return another drawing.

**Reply by tve** [reply]
It might be helpful to write a couple of sentences in the guide about which APIs one should use and which one should avoid. Or mark the blueprint stuff as deprecated...
